doc: add option to disable building dts bindings (#17521)
nrfconnect/sdk-nrf: doc/nrf/dev_model_and_contributions/documentation/build.rst

> .. _testing_versions:
> 
> Optimizing build speed
> **********************
> 
> When working with documentation locally you may enable the ``-DNO_DTS_BINDINGS=ON`` option when configuring the build.
> This option will skip generating devicetree bindings pages, and so speeding up the Zephyr docset build.
> 
> Testing locally
> ***************

Reviewer: ```suggestion

.. _optimizing_doc_build_speed:

Optimizing build speed
**********************

When working with documentation locally, you can enable the ``-DNO_DTS_BINDINGS=ON`` option when configuring the build.
This option skips generating devicetree bindings pages, which speeds up the Zephyr documentation set build process.

.. _testing_versions:

Testing locally
***************
```
